btcpay-setup.sh

```diff
@@ -163,6 +164,7 @@ echo "
 -------SETUP-----------
 Parameters passed:
 BTCPAY_HOST:$BTCPAY_HOST
+BTCPAY_PORT:${BTCPAY_PORT:=80}
```

Comment:
I am wondering if we can have a one liner to not put this line if BTCPAY_PORT is null or unset. It might be confusing for a user using nginx to see `BTCPAY_PORT: 80` when his website is using 443.